prime-router/src/main/kotlin/azure/WorkflowEngine.kt

```diff
@@ -184,7 +184,6 @@ class WorkflowEngine(
         // Send immediately.
         val nextEvent = ReportEvent(Event.EventAction.SEND, reportId, at = null)
         db.transact { txn ->
-            val task = db.fetchAndLockTask(reportId, txn)
```

Comment:
Huh?   Why is this line being eliminated???  As the "atomic read and set" comment above says, the point is to prevent another simultaneous thread/process from processing the same report at the same time.

Reply:
Hi Jim, this val is never used. Seeing that this creates a lock, do you suggest leaving it as is or taking away the <val task => part and just leaving : db.fetchAndLockTask(reportId, txn)?

Reply:
@jimduff-usds @jorg3lopez Kotlin was complaining `task` wasn't being used. maybe we should wrap the variable in synchronized, which then uses the `task` variable as a monitor and then it falls out of scope at the same time as the DB lock?

Reply:
I think just getting rid of "val task =" is sufficient.  The function is being called for its side effect.